### DISCUSSION TITLE
Unable to start client application - npm install throwing error

### DISCUSSION THREAD

**Discussion by JameelCodeMagic** [jasontaylordev/CleanArchitecture#975] [Q&A] [answered]
Have someone faced this issue before? Need help to run the application.
npm install is not working.
Node.js version is 20.5.1.

![image](https://github.com/jasontaylordev/CleanArchitecture/assets/22545635/76ef933a-cc8b-4e34-b09d-d880eb3f4495)

**Comment by JameelCodeMagic** [comment]
Anyone?

**Reply by jasontaylordev** [reply, marked as answer]
Hi @JameelCodeMagic. Please could you try with Node.js LTS?

**Reply by JameelCodeMagic** [reply]
Hi @jasontaylordev,
Node.js LTS did the work.
But why it's not working with Note.js Latest?

Also before trying Node LTS, I tried "yarn install" command and it runs successfully.

Regards,
